packages/aws-cdk-lib/aws-dynamodb/lib/table.ts

```diff
@@ -347,6 +347,7 @@ export interface TableOptions extends SchemaOptions {
   readonly replicationTimeout?: Duration;
 
   /**
+   * [WARNING: Use this flag with caution, misusing this flag may cause deleting existing replicas, refer to the detailed documentation for more information]
```

Comment:
Do the square brackets do something to format this in the dosctring? If not should we remove them like on like 356?

Reply:
to update the summary doc, either we need a paragraph of 20 words, or one sentence that follow this pattern https://github.com/aws/jsii/blob/37082de53576b9c807e83506af336458c6f2c6cb/packages/jsii/lib/docs.ts#L259-L261

that is why I added square brackets here to separate between the warning message, and the actual definition of this flag. I am open for better ideas :)

Reply:
Discussed with @moelasmar and the updated summary line in the table looks good. This is just a workaround to ensure that the warning and the original summary line both show up without the use of any punctuation which would cause the original summary line to be cut off.